### PR TITLE
Fix rendering issue for blogs

### LINKED ISSUE #19532: https://kubernetes.io/blog/2015/04/kubernetes-release-0150/ renders in Chinese
**This is a Bug Report**

**Problem:**
https://kubernetes.io/blog/2015/04/kubernetes-release-0150/ renders in Chinese. I have my user agent preferences set to send `Accept-Language:` with a list of languages not including `zh`.

**Proposed Solution:**
Investigate; check why the Chinese localization seems to be preferred to English.

**Page to Update:**
https://kubernetes.io/blog/2015/04/kubernetes-release-0150/

### PR DESCRIPTION
**Pull request by gavinfish**
Delete url property for old zh posts.

Fix #19532

/cc @sftim

### CODE REVIEW

File: content/zh/blog/_posts/2015-03-00-Weekly-Kubernetes-Community-Hangout.md [hunk 1]
```diff
@@ -80,7 +79,7 @@ Notes from meeting:
     * Can also with careful flag crafting run rsync over this or set up sshd inside container.
 
     * Some feedback via chat:
-    
+
```
Comment (sftim) [inline content/zh/blog/_posts/2015-03-00-Weekly-Kubernetes-Community-Hangout.md]: Did you mean to include this whitespace change @gavinfish?

**Comment by gavinfish** [comment]
> Rather than removing the `url` property, I think it needs `/zh` prepended. Does that sound right?

I have checked the latest zh posts, and they work as expected without url property.

File: content/zh/blog/_posts/2015-03-00-Weekly-Kubernetes-Community-Hangout.md (hunk 1 above)
Comment (gavinfish) [reply content/zh/blog/_posts/2015-03-00-Weekly-Kubernetes-Community-Hangout.md]: Seems that my editor automatically deletes the extra `tab` in the line.

**Comment by xichengliudui** [comment]
/approve
@gavinfish thanks!

**Pull request merged**